mlos_bench/mlos_bench/config/services/azure/arm-templates/azuredeploy-ubuntu-vm.jsonc

```diff
@@ -41,12 +64,11 @@
     },
     "ubuntuOSVersion": {
       "type": "string",
-      "defaultValue": "18.04-LTS",
       "allowedValues": [
-        "12.04.5-LTS",
-        "14.04.5-LTS",
         "16.04.0-LTS",
-        "18.04-LTS"
+        "18.04-LTS",
+        "20.04-LTS",
+        "22.04-LTS"
```

Comment:
As discussed elsewhere, these won't work atm due to how the osOffers changed, but we can fix it later too.